tests/server-integration/Utils.hs

```diff
@@ -164,7 +179,7 @@ executeCommand method urlParts body proxy options =
       body
       proxy
       ( mconcat
-          [ header "token" "root"
+          [ header "coffer-backend" makeCofferHeader
```

Comment:
Basically I haven't made tests for incorrect headers. But it's would make more sense to add them with #118 issue